src/gui/editors/AutomationEditor.cpp

```diff
@@ -1044,7 +1044,7 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 			for( int i = 0; i < 2; ++i )
 			{
 				const QString & label = m_pattern->firstObject()
-						->displayValue( level[i] );
+						->displayValue( level[i] /100 );
```

Comment:
This should also use a space after the slash, like the other lines, for consistency: `/ 100`